Tests/UIKitTests/UIViewControllerExtensionsTests.swift

```diff
@@ -120,5 +120,22 @@ final class UIViewControllerExtensionsTests: XCTestCase {
         XCTAssertNil(childViewController.parent)
     }
 
+    func testPresentPopover() {
```

Comment:
Hey @marcocapano :))
Seems like its falling on tvOS, I think you should also constraint the tests to #if os(iOS) 👍

Reply:
You're right! Did it for the source but not for the test

Reply:
What do you think of this? I would really appreciate a suggestion 😊🙏🏻
>  If I add another test for example to check that the size of the popover is the one set in the parameter, should I still test other things like presentedViewController, presentingViewController etc.. that I already test in the other test?

Reply:
Yes, it ensures that you have consistent behaviour in your function.

Reply:
For sure 👍

Reply:
Done, thanks for your help!